resources/styles/global/typography.less

```diff
@@ -41,6 +41,8 @@
     font-style: normal;
     font-size: @size;
     line-height: @line-height;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
Right, but in this case, the default value was dropped. I added it, even if it doesn't do anything. It's been removed from the specification and is currently not on the standard track, but who knows?